src/Features/Core/Completion/ICompletionService.cs

```diff
@@ -12,6 +12,11 @@ namespace Microsoft.CodeAnalysis.Completion
     internal interface ICompletionService : ILanguageService
     {
         /// <summary>
+        /// The language that this <see cref="ICompletionService"/> targets.
+        /// </summary>
+        string LanguageName { get; }
```

Comment:
Yeah, I'm still trying to get rid of it. It gets used way down the line by snippets to determine what language to grab an ```ISnippetInfoService``` for. I still need to dig into that. If I can figure out the right way to get the language name in that case, I'll kill this property.